code/game/machinery/cryopod.dm

```diff
@@ -0,0 +1,455 @@
+		if(istype(O,/datum/objective/mutiny) && O.target == mob_occupant.mind)
+			qdel(O)
+		else if(O.target && istype(O.target, /datum/mind))
+			if(O.target == mob_occupant.mind)
+				for(var/datum/mind/M in O.get_owners())
+					to_chat(M.current, "<BR><span class='userdanger'>You get the feeling your target is no longer within reach. Time for Plan [pick("A","B","C","D","X","Y","Z")]. Objectives updated!</span>")
+				O.target = null
+				spawn(10) //This should ideally fire after the occupant is deleted.
+					if(!O)
+						return
+					O.find_target()
+					O.update_explanation_text()
+					if(!(O.target))
+						O.owner.objectives -= O
+						qdel(O)
+					for(var/datum/mind/M in O.get_owners())
+						M.announce_objectives()
+
+	if(mob_occupant.mind && mob_occupant.mind.assigned_role)
+		//Handle job slot/tater cleanup.
+		var/job = mob_occupant.mind.assigned_role
+		SSjob.FreeRole(job)
+		if(mob_occupant.mind.objectives.len)
+			mob_occupant.mind.objectives.Cut()
+			mob_occupant.mind.special_role = null
+
+	// Delete them from datacore.
+
+	var/announce_rank = null
+	for(var/datum/data/record/R in GLOB.data_core.medical)
+		if((R.fields["name"] == mob_occupant.real_name))
+			qdel(R)
+	for(var/datum/data/record/T in GLOB.data_core.security)
+		if((T.fields["name"] == mob_occupant.real_name))
+			qdel(T)
+	for(var/datum/data/record/G in GLOB.data_core.general)
+		if((G.fields["name"] == mob_occupant.real_name))
+			announce_rank = G.fields["rank"]
+			qdel(G)
+
+	for(var/obj/machinery/computer/cloning/cloner in world)
+		for(var/datum/data/record/R in cloner.records)
+			if(R.fields["name"] == mob_occupant.real_name)
+				cloner.records.Remove(R)
+
+	//Make an announcement and log the person entering storage.
+	if(control_computer)
+		control_computer.frozen_crew += "[mob_occupant.real_name]"
+
+	if(GLOB.announcement_systems.len)
+		var/obj/machinery/announcement_system/announcer = pick(GLOB.announcement_systems)
+		announcer.announce("CRYOSTORAGE", mob_occupant.real_name, announce_rank, list())
+		visible_message("<span class='notice'>\The [src] hums and hisses as it moves [mob_occupant.real_name] into storage.</span>")
+
+
+	for(var/obj/item/W in mob_occupant.GetAllContents())
+		if(W.loc.loc && (( W.loc.loc == loc ) || (W.loc.loc == control_computer)))
+			continue//means we already moved whatever this thing was in
+			//I'm a professional, okay
+		for(var/T in preserve_items)
+			if(istype(W, T))
+				if(control_computer && control_computer.allow_items)
+					control_computer.frozen_items += W
+					mob_occupant.transferItemToLoc(W, control_computer, TRUE)
+				else
+					mob_occupant.transferItemToLoc(W, loc, TRUE)
+
+	for(var/obj/item/W in mob_occupant.GetAllContents())
+		qdel(W)//because we moved all items to preserve away
+		//and yes, this totally deletes their bodyparts one by one, I just couldn't bother
+
+	if(iscyborg(mob_occupant))
+		var/mob/living/silicon/robot/R = occupant
+		if(!istype(R)) return ..()
```

Comment:
This is the part that is causing the error. `return ..()` means return the parent proc, except there is no parent because this is declaring the proc.

```suggestion
		if(!istype(R)) return
```

Reply:
Done and done, thanks